components/network/openssh/files/ssh-askpass

```diff
@@ -0,0 +1,31 @@
+#! /usr/bin/ksh
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright 2009 Sun Microsystems, Inc.  All rights reserved.
+# Use is subject to license terms.
+#
+
+prompt=$(echo $1 | sed s/_/__/g)
+ICON=/usr/share/pixmaps/blueprint-keyring.png
+exec /usr/bin/zenity --entry --title "ssh-askpass" \
+    --text="$prompt" --hide-text --window-icon=$ICON
```

Comment:
Hi, while there is merit in both proposals, one is cosmetic and another is
an improvement without which this variant of askpass lived for years as
part of SunSSH. (I tried to build a few others, including contribs that go
with openssh-portable, and the old x11-ssh-askpass from 2004 or so - they
didn't work on Hipster out of the box).

I suggest to solve problems one at a time, and for now - quickly and simply
restore the 'status quo' (return the useful and habitual functionality into
the world of living) by merging this PR if there are no other objections. I
am on vacation, so someone else (or myself much later) can tackle other
bits of evolution in separate PRs.

Jim
--
Typos courtesy of GMail on my Samsung Android

26.08.2016 7:11 пользователь "Alexander Pyhalov" <notifications@github.com>
написал:

> In components/network/openssh/files/ssh-askpass
> <https://github.com/OpenIndiana/oi-userland/pull/2365#discussion_r76364979>
> :
>
> > +# If applicable, add the following below this CDDL HEADER, with the
> > +# fields enclosed by brackets "[]" replaced with your own identifying
> > +# information: Portions Copyright [yyyy] [name of copyright owner]
> > +#
> > +# CDDL HEADER END
> > +#
> > +
> > +#
> > +# Copyright 2009 Sun Microsystems, Inc.  All rights reserved.
> > +# Use is subject to license terms.
> > +#
> > +
> > +prompt=$(echo $1 | sed s/_/__/g)
> > +ICON=/usr/share/pixmaps/blueprint-keyring.png
> > +exec /usr/bin/zenity --entry --title "ssh-askpass" \
> > +    --text="$prompt" --hide-text --window-icon=$ICON
>
> Should it have some fallback when zenity is not found?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenIndiana/oi-userland/pull/2365/files/32ad0332ec68da1cc613113981f2831980d44889#r76364979>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABj5lMyUagaPjipv5AhPYL3N6LYv3Bfyks5qjnV2gaJpZM4JtprB>
> .
>